### PR TITLE
Created payment component

### PR DESCRIPTION
**Pull request by princegeorge121**

### CODE REVIEW

File: client/mycart/src/app/admin/admin-routing.module.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const routes: Routes = [
     path: '', component: AdminLayoutComponent, children: [
       { path: 'product/create', component: AddProductComponent },
       { path: 'feedback/view', component: ViewFeedbackComponent },
-      { path: 'product/update', component:UpdateProductComponent}
+      { path: 'product/update', component:UpdateProductComponent},
```
Comment (mishelshaji) [inline client/mycart/src/app/admin/admin-routing.module.ts]: Did you miss the product id? Also, format the code.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/mycart/src/app/admin/admin-routing.module.ts [hunk 2]
```diff
@@ -9,9 +9,9 @@ import { ViewFeedbackComponent } from './view-feedback/view-feedback.component';
 const routes: Routes = [
   {
     path: '', component: AdminLayoutComponent, children: [
-      { path: 'product/create', component: AddProductComponent },
+      { path: 'products/create', component: AddProductComponent },
       { path: 'feedback/view', component: ViewFeedbackComponent },
-      { path: 'product/update', component:UpdateProductComponent},
+      { path: 'products/update', component: UpdateProductComponent },
```
Comment (mishelshaji) [inline client/mycart/src/app/admin/admin-routing.module.ts]: What about the id?

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/mycart/src/app/admin/admin-routing.module.ts (hunk 1 above)
Comment (princegeorge121) [reply client/mycart/src/app/admin/admin-routing.module.ts]: Done. Product ID is auto generated, so not updating it.

File: client/mycart/src/app/admin/admin-routing.module.ts (hunk 2 above)
Comment (princegeorge121) [reply client/mycart/src/app/admin/admin-routing.module.ts]: All fields except id is updatable.

Comment (princegeorge121) [reply client/mycart/src/app/admin/admin-routing.module.ts]: Done.

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**